### PR TITLE
Added mxwcli functions : base on query commands

### PR DESCRIPTION
**Pull request by githubckgoh1439**
This commit along with features been added :

1. mxwcli query module-maintainer address base on Issuer, Provider, Middleware, Fee-collector
2. mxwcli query FT/NFT fee-setting base on symbols

### CODE REVIEW

**Review by b00f — APPROVED** [review]

**Comment by githubckgoh1439** [comment]
Features of 'mxwcli query FT/NFT fee-setting base on symbols' need to be fixed later: 

1. Able to include different type of token-action, eg. transfer, mint, bur, transferOwnership, acceptOwnership...etc

**Review by b00f — APPROVED** [review]

File: x/fee/client/cli/query.go [hunk 1]
```diff
@@ -71,12 +74,18 @@ func GetFungibleTokenFeeSetting(cdc *codec.Codec) *cobra.Command {
 
 			symbol := args[0]
 
-			res, _, err := cliCtx.QueryWithData(fmt.Sprintf("custom/%s/%s/%s", "fee", fee.QueryFungibleTokenFeeSetting, symbol), nil)
+			res, _, err := cliCtx.QueryWithData(fmt.Sprintf("custom/%s/%s/%s", "token", fungible.QueryTokenData, symbol), nil)
 			if err != nil {
 				fmt.Printf("Could not get fungible token fee setting: %s\n", err)
```
Comment (yenkhoon) [inline x/fee/client/cli/query.go:79]: @githubckgoh1439 fix the error message.

File: x/fee/client/cli/query.go [hunk 2]
```diff
@@ -96,7 +105,13 @@ func GetNonFungibleTokenFeeSetting(cdc *codec.Codec) *cobra.Command {
 
 			symbol := args[0]
 
-			res, _, err := cliCtx.QueryWithData(fmt.Sprintf("custom/%s/%s/%s", "fee", fee.QueryNonFungibleTokenFeeSetting, symbol), nil)
+			res, _, err := cliCtx.QueryWithData(fmt.Sprintf("custom/%s/%s/%s", "nonFungible", nonfungible.QueryTokenData, symbol), nil)
+			if err != nil {
+				fmt.Printf("Could not get non-fungible token fee setting: %s\n", err)
```
Comment (yenkhoon) [inline x/fee/client/cli/query.go:110]: @githubckgoh1439 fix the error message.

**Review by yenkhoon — CHANGES_REQUESTED** [review]
@githubckgoh1439 improve the error message, the error message is wrong. It should tell the user that the token is not found.

File: x/fee/client/cli/query.go (hunk 1 above)
Comment (githubckgoh1439) [reply x/fee/client/cli/query.go]: Fixed the message using 'fungible token not exist'.

File: x/fee/client/cli/query.go (hunk 2 above)
Comment (githubckgoh1439) [reply x/fee/client/cli/query.go]: Fixed the message using 'non-fungible token not exist'.

File: x/fee/client/cli/query.go [hunk 3]
```diff
@@ -76,12 +76,12 @@ func GetFungibleTokenFeeSetting(cdc *codec.Codec) *cobra.Command {
 
 			res, _, err := cliCtx.QueryWithData(fmt.Sprintf("custom/%s/%s/%s", "token", fungible.QueryTokenData, symbol), nil)
 			if err != nil {
-				fmt.Printf("Could not get fungible token fee setting: %s\n", err)
+				fmt.Printf("This fungible token does not exist\n")
```
Comment (yenkhoon) [inline x/fee/client/cli/query.go:79]: No nid this, please add in the symbol that user keyed in.

File: x/fee/client/cli/query.go [hunk 4]
```diff
@@ -107,7 +107,7 @@ func GetNonFungibleTokenFeeSetting(cdc *codec.Codec) *cobra.Command {
 
 			res, _, err := cliCtx.QueryWithData(fmt.Sprintf("custom/%s/%s/%s", "nonFungible", nonfungible.QueryTokenData, symbol), nil)
 			if err != nil {
-				fmt.Printf("Could not get non-fungible token fee setting: %s\n", err)
+				fmt.Printf("This non-fungible token does not exist\n")
```
Comment (yenkhoon) [inline x/fee/client/cli/query.go:110]: No nid this, please add in the symbol that user keyed in.

**Review by yenkhoon — CHANGES_REQUESTED** [review]
Please make the error message return the symbol entered by user.

File: x/fee/client/cli/query.go (hunk 4 above)
Comment (yenkhoon) [reply x/fee/client/cli/query.go:110]: @githubckgoh1439

Comment (githubckgoh1439) [reply x/fee/client/cli/query.go]: Updated the Error message along with enter-symbol.

Comment (yenkhoon) [reply x/fee/client/cli/query.go]: @githubckgoh1439 remove this please.

Comment (githubckgoh1439) [reply x/fee/client/cli/query.go]: Updated the Error message along with enter-symbol for non-fungible token.

Comment (githubckgoh1439) [reply x/fee/client/cli/query.go]: This updated:
https://github.com/maxonrow/maxonrow-go/pull/200/commits/e45c12e9b7a091d4e75d53460bbb632df252f9f0

File: x/fee/client/cli/query.go [hunk 5]
```diff
@@ -76,7 +76,7 @@ func GetFungibleTokenFeeSetting(cdc *codec.Codec) *cobra.Command {
 
 			res, _, err := cliCtx.QueryWithData(fmt.Sprintf("custom/%s/%s/%s", "token", fungible.QueryTokenData, symbol), nil)
 			if err != nil {
-				fmt.Printf("This fungible token does not exist\n")
+				fmt.Printf("This fungible token does not exist: %s\n", symbol)
```
Comment (yenkhoon) [inline x/fee/client/cli/query.go:79]: @githubckgoh1439 "This" is still there.

**Review by yenkhoon — CHANGES_REQUESTED** [review]

File: x/fee/client/cli/query.go (hunk 5 above)
Comment (githubckgoh1439) [reply x/fee/client/cli/query.go]: Fixed under this: 
https://github.com/maxonrow/maxonrow-go/pull/200/commits/28a37e076e1f52dda17d3bf53755b2ab281bc3e7

File: x/fee/client/cli/query.go (hunk 4 above)
Comment (githubckgoh1439) [reply x/fee/client/cli/query.go]: Fixed under this: 
https://github.com/maxonrow/maxonrow-go/pull/200/commits/28a37e076e1f52dda17d3bf53755b2ab281bc3e7

**Review by yenkhoon — APPROVED** [review]

**Pull request merged**